### DISCUSSION TITLE
Name of an Connected Window should use the name of the connection

### DISCUSSION THREAD

**Discussion by Gunnar-GO** [rustdesk/rustdesk#2815] [Ideas]
At the moment the connections uses the numbers of the connection. 
In the connection / desktop window you can name the connections so it is far more human understandable.

More easy would it be if the remote window would also uses this. With more than one connection open you have to look every time which window is which connection.

**Comment by youhers** [comment]
This suggestion is good, I hope the author will adopt it later, it is estimated that the author does not have time to engage in it at present

**Comment by Gunnar-GO** [comment]
Actually i don't know why my post is now an discussion, i started it as [Feature Request] in Issues.

At the moment i don't recognise any movement or notice. So will see.

**Comment by youhers** [comment]
It is estimated that the author is very busy. At present, there are still many bugs to be fixed in the software. It is estimated that it will be considered when it is stable later. I also submitted the author without reply

**Comment by rustdesk** [comment]
@fufesou for multiple tabs, always use the nane of active connection.

**Reply by Gunnar-GO** [reply]
i assume the answer is for another window`?

**Comment by fufesou** [comment]
Rustdesk shows the alias of the peer.

But only rename on main page for now.
Renaming on remote page is not supported.



https://user-images.githubusercontent.com/13586388/213913549-7fd206ae-bfeb-4ba4-ba71-3404a6d78556.mp4

**Comment by rustdesk** [comment]
i guess they mean the name shown in windows taskbar.

**Reply by Kingtous** [reply]
If so, we need to listen tabbar switch event, and set the window title when its changed.

**Reply by rustdesk** [reply]
yes

**Comment by youhers** [comment]
It would be great if it could be modified

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/2920 fixed
